base/libgit2/config.jl

```diff
@@ -1,5 +1,12 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    GitConfig(path::AbstractString, level::Consts.GIT_CONFIG = Consts.CONFIG_LEVEL_APP, force::Bool=false)
```

Comment:
Inconsistent use of spaces around `=`, here and in the other signatures. I'd suggest removing the spaces since that seems to be more common in Base docstrings.